BaseCode/App/UI/Screens/CreateProfile/CreateProfileViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  CreateProfileViewController.swift
+//  BaseCode
+//
+//  Created by BrainX IOS Dev on 25/08/2023.
+//
+
+import UIKit
+
+class CreateProfileViewController: ViewController<CreateProfileViewModel> {
+
+    //MARK: - Outlets
+    
+    @IBOutlet var profileView: CreateProfileView!
+    
+    //MARK: - Lifecycle Methods
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        initialSetup()
+        addTargets()
+    }
+    
+    override func setupBinding() {
+        viewModel
+            .$avatar
+            .sink { [weak self] avatar in
+                self?.profileView.avatorImageView.image = avatar?.image ?? UIImage(named: Images.profilePlaceholder)
+            }
+            .store(in: &bag)
+        viewModel
+            .$currentNameCount
+            .sink { [weak self] count in
+                guard let self = self else {
+                    return
+                }
+                self.profileView.setupNameField(
+                    withName: self.viewModel.userName,
+                    maxCount: self.viewModel.maxNameCount
+                )
+            }
+            .store(in: &bag)
+        viewModel
+            .$age
+            .sink { [weak self] age in
+                if age > 0 {
+                    self?.profileView.ageTextField.text = String(age)
+                }
+            }
+            .store(in: &bag)
+        viewModel
+            .$gender
+            .sink { [weak self] gender in
+                self?.profileView.setupGenderView(gender)
+            }
+            .store(in: &bag)
+    }
+    
+    //MARK: - Action Methods
+    
+    @IBAction
+    func backButtonTapped(_ sender: Any) {
+    }
+    
+    @IBAction
+    func addAvatorButtonTapped(_ sender: Any) {
+        viewModel.handleAddAvatarTapped()
+    }
+    
+    @IBAction
+    func nameTextFieldChanged(_ textField: UITextField) {
+        viewModel.userName = textField.text ?? AppConstants.empty
+    }
```

Comment:
Suggestion: 
Look into how you can use combine to bind text field to a property.